apps/webservice/src/app/api/v1/jobs/[jobId]/route.ts

```diff
@@ -58,7 +59,16 @@ export const GET = async (
     jobVariableRows.map((v) => [v.key, v.value]),
   );
 
-  return NextResponse.json({ ...je.job, ...je, variables });
+  const jobTargetMetadataRows = await db
+    .select()
+    .from(targetMetadata)
+    .where(eq(targetMetadata.targetId, je.target?.id ?? ""));
+
+  const metadata = Object.fromEntries(
+    jobTargetMetadataRows.map((m) => [m.key, m.value]),
```

Comment:
does github support `/`? maybe we shoud replace with `_`

Reply:
Yah, we need to just make it _ I think.